pkg/cluster/install.go

```diff
@@ -22,13 +22,13 @@ import (
 
 const (
 	kubectlCmd = "kubectl"
-	// profiles bundles ready to be installed files under `prepare`. The rest of the resources
+	// profiles bundles ready to be installed files under `install`. The rest of the resources
 	// are left for manual configuration.
-	prepareManifestFile = "prepare.yaml"
+	installManifestFile = "prepare.yaml"
```

Comment:
this could also be renamed to `install.yaml` but this might be too nit. wdyt about this?

Reply:
can't do this until we cut a `profiles` release with the installation file called `install.yaml`. Going to do this separately :smile: